.github/workflows/Build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         path: shasta-build/shasta-Ubuntu-20.04/bin/shasta
     - uses: actions/upload-artifact@master
       with:
-        name: shasta-Ubuntu-20.04.tar
+        name: shasta-Linux.tar
```

Comment:
I would keep the Ubuntu version in the name of the tar file because that is not necessarily portable to other Ubuntu versions.

Reply:
This archive has `bin`, `docs`, `docker` & `conf` directories. `bin` has Python3 scripts and the Shasta binary, which works on Ubuntu 20.04 LTS and Ubuntu 18.04 LTS. 

If we name the executable `shasta-Linux`, then we should name this `shasta-Linux.tar`, no?

Reply:
No, because the tar file contains the dynamic executable and shared library (Python module), which are not portable across Ubuntu versions. So for example if you try to use the Python API using a 20.04 build on 18.04, it will not work. If you want to use the Python API and you don't want to build the code yourself, you have to download the tar file that was built on the Ubuntu version you are using. For this reason I think we should keep the Ubuntu version in the name of tar files.

For the static executable it's different because it is very portable not just across Ubuntu, but also to other Linux versions.

Conceivably, we could add a portable tar file than contains everything but the dynamic library, the dynamic executable, and the Python API scripts. The `bin` directory would probably only contain the static executable, and perhaps a few Python scripts that don't use `'import shasta`'. This tar file could be given a generic name that does not include the Ubuntu version.

Reply:
Good point about the dynamic libraries.